posts/what-is-machine-learning.md

```diff
@@ -0,0 +1,22 @@
+---
+authors:
+- Jeannine Stark
+categories:
+- Cloud
+- Language
+date: 2018-04-27T23:45:52.000Z
+title: "What is Machine Learning?"
+image: 
+---
+
+## What is Machine Learning?
+
+Machine Learning is another aspect of the Big Data ecosystem as well as a type of Artificial Intelligence. More about what Big Data is can be found [in a previous blog entry ](https://blog.ippon.tech/what-is-big-data/). Artificial Intelligence is capabilities that are computers that act or perform human like tasks. One primitive version of Artificial Intelligence that has been around for decades are computer card games. In this case a computer is programmed to behave like a human, making decisions on the card to play based on the situation. As machines got more powerful Machine Learning became a subcategory of Artificial Intelligence. In the card game the program had explicit rules for what to do in a situation whereas Machine Learning is where the program determines what to do based on prior data. The program determines what to do based on its training to produce a model. The training aspect of the machine learning is giving the computer enough data to make determinations. There is no magic amount of data that will provide the correct answer. In the card example the training would be done off data about what moves the prior player made. The output of the training is called the model. The model tells the computer what moves to make based on the situation in the game being played. There are two different ways to train a machine learning model, supervised and unsupervised learning.
+
+In supervised learning the Machine Learning model is trained based on previously captured data and the known output. One of the most basic use cases I have seen for this is document classification. The training data would be a set of documents with the classification. In the data set, Document A would be the input and medical document would be the classification, Document B would be the input and the classification would be a Credit Card statement and so on. The model would be trained based on this information so when a new document is received the type of document can be determined.
```

Comment:
Put quotes around `medical document` and `Credit Card statement ` to indicate they are results